migrator/migrations/m_196_to_m_197_set_poduid_where_null/migration_impl.go

```diff
@@ -0,0 +1,241 @@
+package m196tom197
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/stackrox/rox/generated/storage"
+	listeningEndpointsSchema "github.com/stackrox/rox/migrator/migrations/m_196_to_m_197_set_poduid_where_null/schema/listening_endpoints"
+	podSchema "github.com/stackrox/rox/migrator/migrations/m_196_to_m_197_set_poduid_where_null/schema/pods"
+	processIndicatorSchema "github.com/stackrox/rox/migrator/migrations/m_196_to_m_197_set_poduid_where_null/schema/process_indicators"
+	podDatastore "github.com/stackrox/rox/migrator/migrations/m_196_to_m_197_set_poduid_where_null/store/pod"
```

Comment:
The name here is incorrect. I am not sure what it should be called. I should call it `podStore`, but I have a variable called that.